Client/Frontend/Browser/Tab.swift

```diff
@@ -527,6 +531,15 @@ private class TabWebView: WKWebView, MenuHelperInterface {
         }
     }
 
+    @objc func swizzledMenuHelperFindInPage() {
+        if let tabWebView = superview?.superview as? TabWebView {
+            tabWebView.evaluateJavaScript("getSelection().toString()") { result, _ in
```

Comment:
Can the swizzled method see the original?

```swift
if let tabWebView = superview?.superview as? TabWebView {
    tabWebView.menuHelperFindInPage()
}
```

Reply:
No because `self` in this method actually points to a `WKContentView`.